### PR TITLE
streaming search: redesigned error/skipped items popover

### PR DESCRIPTION
**Pull request by limitedmage**
- Redesigned popover to match [Figma](https://www.figma.com/file/IyiXZIbPHK447NCXov0AvK/13928-Streaming-search?node-id=809%3A28574)
- Items are now collapsible
- The first item in the list starts open (unless its severity is `info`)

image

https://user-images.githubusercontent.com/206864/104520137-770e5d80-55af-11eb-8ee2-a80243c63492.mov

### CODE REVIEW

**Comment by keegancsmith** [comment]
https://user-images.githubusercontent.com/187831/104557145-67dbee00-5649-11eb-813b-9d88f9128e9d.mp4

File: client/web/src/search/results/streaming/progress/StreamingProgressSkippedPopover.tsx [hunk 1]
```diff
@@ -27,6 +31,52 @@ const sortBySeverity = (a: Skipped, b: Skipped): number => {
...
+            onClick={toggleIsOpen}
+            disabled={!skipped.message}
+        >
+            <h4 className="d-flex align-items-center mb-0 w-100">
+                {skipped.severity === 'info' ? (
+                    <InformationOutlineIcon className="icon-inline mr-2 streaming-skipped-item__icon flex-shrink-0" />
+                ) : (
+                    <AlertCircleIcon className="icon-inline mr-2 streaming-skipped-item__icon flex-shrink-0" />
+                )}
+                <span className="flex-grow-1 text-left">{skipped.title}</span>
```
Comment (lguychard) [inline client/web/src/search/results/streaming/progress/StreamingProgressSkippedPopover.tsx:57]: Should we `upperFirst` the title to make sure it's sentence case?

**Review by lguychard — APPROVED** [review]
Awesome!

File: client/web/src/search/results/streaming/progress/StreamingProgressSkippedPopover.tsx (hunk 1 above)
Comment (limitedmage) [reply client/web/src/search/results/streaming/progress/StreamingProgressSkippedPopover.tsx:57]: I think we should instead ensure that the titles come correctly formatted on the backend. I'd like to do as little front-end manipulation of these as possible. We could do this as part of the error message QA PR #17171

**Pull request merged**